server/routes/reviewPlan/index.ts

```diff
@@ -24,10 +24,10 @@ export default function reviewPlanRoutes(router: Router) {
   const exemptionReasonController = new ExemptionReasonController()
   const confirmExemptionController = new ConfirmExemptionController()
 
-  router.use('/plan/:prisonNumber/review/exemption/**', [
+  router.use('/plan/:prisonNumber/review/exemption', [
     checkPrisonIsEnabled(),
     checkUserHasEditAuthority(),
-    checkExemptionExistsInPrisonerContext,
+    createEmptyReviewExemptionDtoIfNotInPrisonerContext,
```

Comment:
If on the create exemption page or create review page, create an empty exemption or review Dto. If on any exemption or review page that isn't the first one in the respective journey, check for a Dto. (The Dto check now includes a check for the exemption Dto as well as the review Dto).